salt/modules/status.py

```diff
@@ -1050,11 +1050,16 @@ def diskusage(*args):
     # query the filesystems disk usage
     ret = {}
     for path in selected:
-        fsstats = os.statvfs(path)
-        blksz = fsstats.f_bsize
-        available = fsstats.f_bavail * blksz
-        total = fsstats.f_blocks * blksz
-        ret[path] = {"available": available, "total": total}
+        if os.path.exists(path):
```

Comment:
There are changes to the tests bu it doesn't look like any validate this work.

Reply:
Correct. I'll cover this on the tests

Reply:
@dwoz I've added tests for covering this. I guess this is ready for another review. Thanks!